File: clexo4.py
def max_de_trois(a, b, c):
    maximum = a
    if b > maximum:
        maximum = b
    if c > maximum:
        maximum = c
    return maximum

n1 = float(input("Entrez le 1er nombre : "))
n2 = float(input("Entrez le 2ème nombre : "))
n3 = float(input("Entrez le 3ème nombre : "))

resultat = max_de_trois(n1, n2, n3)
print(f"Le maximum est {resultat}")

File: clexo4.js
function maxDeTrois(a, b, c) {
let maximum = a;
if (b > maximum) {
maximum = b;
}
if (c > maximum) {
maximum = c;
}
return maximum;
}

let num1 = parseFloat(prompt("Entrez le 1er nombre : "));
let num2 = parseFloat(prompt("Entrez le 2ème nombre : "));
let num3 = parseFloat(prompt("Entrez le 3ème nombre : "));

let resultat = maxDeTrois(num1, num2, num3);
console.log("Le maximum est " + resultat);